### PR TITLE
Adds UI for admin settings: microservices and analytics

### LINKED ISSUE #202: Add UI for admin settings : microservices and analytics
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

image
image

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds UI for : 
* `admin/settings/microservices`
* `admin/settings/analytics`

#### Changes proposed in this pull request:

image
image

Two commits for two separate routes

Fixes #202

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
Deployment link: https://cc96-opev-frontend.herokuapp.com/admin/settings/microservices
@geekyd Please review

File: app/templates/components/forms/admin/settings/microservices-form.hbs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <h3 class="ui header">
+    {{t 'Android App'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/microservices-form.hbs]: `Android App Generator`

File: app/templates/components/forms/admin/settings/microservices-form.hbs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  </h3>
+  <div class="field">
+    <label>
+      {{t 'URL'}}
+    </label>
+    {{input type='url' name='android_app' placeholder='http://droidgen.eventyay.com/api/v2/'}}
+  </div>
+  <h3 class="ui header">
+    {{t 'Web App'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/microservices-form.hbs]: `Web App Generator`

File: app/templates/components/forms/admin/settings/microservices-form.hbs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <h3 class="ui header">
+    {{t 'Android App'}}
+  </h3>
+  <div class="field">
+    <label>
+      {{t 'URL'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/microservices-form.hbs]: `API Server`

File: app/templates/components/forms/admin/settings/microservices-form.hbs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    </label>
+    {{input type='url' name='android_app' placeholder='http://droidgen.eventyay.com/api/v2/'}}
+  </div>
+  <h3 class="ui header">
+    {{t 'Web App'}}
+  </h3>
+  <div class="field">
+    <label>
+      {{t 'URL'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/microservices-form.hbs]: `API Server`

Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/microservices-form.hbs]: @niranjan94 done.

**Comment by CosmicCoder96** [comment]
@niranjan94 @geekyd I have done the changes, please review.

File: app/templates/components/forms/admin/settings/analytics-form.hbs [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <h3 class="ui header">
+    {{t 'Google Analytics Tracking ID'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/analytics-form.hbs]: `Google Analytics`

**Comment by CosmicCoder96** [comment]
@niranjan94 Done.

**Pull request merged**